docs/product-user-guide/src/main/asciidoc/chap-deploying-projects.adoc

```diff
@@ -26,71 +26,57 @@ You can set container aliases when creating containers in the Intelligent Proces
 
 To edit the alias value before building your project in Business Central:
 
-. In Business Central, click *Authoring* -> *Project Authoring*.
-. In the *Project Explorer* window, click *Open Project Editor*.
-. On the title bar, click *Build* -> *Build & Deploy*.
+. In Business Central, select *Menu* -> Design -> *Projects*.
```

Comment:
shouldnt it be *Design*?

Reply:
That's actually on purpose but the subject of a pendantic discussion to be had later. I'll make bold for now for expediancy  ;-)